workflow-bot-app/src/main/java/com/symphony/bdk/workflow/engine/camunda/bpmn/CamundaBpmnBuilder.java

```diff
@@ -110,7 +109,12 @@ private void checkParentIsKnown(Map<String, String> parentActivities, String wor
   }
 
   private BpmnModelInstance workflowToBpmn(Workflow workflow) throws JsonProcessingException {
-    ProcessBuilder process = Bpmn.createExecutableProcess(createUniqueProcessId(workflow)).name(workflow.getId());
+    String processId = "";
+    if (workflow.getId() != null) {
+      processId = workflow.getId().replaceAll("\\s+", "_");
+    }
+
+    ProcessBuilder process = Bpmn.createExecutableProcess(processId).name(workflow.getId());
```

Comment:
we can add a unit test for the workflow update, deploying 2 different workflows with the same id and checking that it is the latest deployed one that is executed